src/components/RepoCard/index.module.css

```diff
@@ -0,0 +1,26 @@
+.card {
```

Comment:
Just going to leave a general message on the whole of the CSS:

Style is very close to GitHub, set out very nicely and user friendly
Navbar is naughty :)

Maybe a bit of white space between the searchbar and search button

Reply:
Yes, good. Done.